operators/test/e2e/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Nit: we used to use `#!/usr/bin/env bash` and add a blank line between the shebang and the license header.